### DISCUSSION TITLE
For new Contributors

### DISCUSSION THREAD

**Discussion by SarthakKeshari** [SarthakKeshari/Java-Questions-and-Solutions#24] [Announcements]
Hello Contributors,
Some issues have been tagged as good first issue. You may ask for assigning those issue to you if you're beginner to open source contribution.

**Comment by ghost** [comment]
Hello sir ,can I directly start Contribute without raising Issues?

**Reply by SarthakKeshari** [reply]
Yes, It is necessary to create an issue or claim an issue which has no assignees. Create an issue and add a question of your choice. Make sure question that you add is a bit tricky or conceptual.
I will assign you your issue then if I get your comment as "Kindly assign me this issue"

**Reply by ghost** [reply]
Okay sir ,Thank you
